library/globals.inc.php

```diff
@@ -2518,10 +2304,125 @@
        '30',
       xl('Rx Bottom Margin (px)')
     ),
+    
+    'erx_enable' => array(
+      xl('Enable NewCrop eRx Service'),
+      'bool',
+      '0',
+      xl('Enable NewCrop eRx Service.') + ' ' +
+      xl('Contact Medical Information Integration, LLC at http://mi-squared.com or ZH Healthcare at http://zhservices.com for subscribing to the NewCrop eRx service.')
```

Comment:
how about we just say "Contact the community for information on subscribing to the NewCrop eRx service."